api/versions.go

```diff
@@ -341,8 +343,24 @@ func (api *DatasetAPI) putVersion(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Populate distributions (static only)
+	// Populate distributions & validate spaces in IDs(static only)
 	if version.Type == models.Static.String() {
```

Comment:
There is already a check for static type on line 371, could this section be moved in there to avoid the duplicate if statements?